emulator/iSkipper/iClickerRadio.h

```diff
@@ -6,7 +6,8 @@
 #include "RFM69.h"
 
 // threshold for triggerins fifo transmit interrupt
```

Comment:
If you make the above change to `PAYLOAD_LENGTH_RECV` can you undo this? I have no real clue how this `RF_FIFOTHRESH_TXSTART_FIFOTHRESH_IC` thing works, and I basically just blindly copied it when I forked the original RFM69 library.

Reply:
`RF_FIFOTHRESH_TXSTART_FIFOTHRESH_IC` makes it so that the RF transmitter will wait until `n+1` bytes are placed in the FIFO before transmitting. The more sane solution is to use `RF_FIFOTHRESH_TXSTART_FIFONOTEMPTY` which just transmits the bytes placed in the FIFO after you unsleep the radio (which is what `send` does)

Reply:
Changing to `RF_FIFOTHRESH_TXSTART_FIFONOTEMPTY` sounds more like what we want, but I don't want to change it to this unless someone can test it on a base station and make sure everything still works.

Reply:
Yeah, I'm wary about the testing part as well, let's see if someone with a base station can test it as part of the welcome packet PR.